src/pages/EditProfile.js

```diff
@@ -1,32 +1,11 @@
 import React, { Component } from 'react';
-import { AsyncStorage, StyleSheet,Text,TextInput, View,Image,TouchableOpacity, KeyboardAvoidingView } from 'react-native';
+import firebase from 'firebase';
+import {StyleSheet,Text,TextInput, View,Image,TouchableOpacity, KeyboardAvoidingView } from 'react-native';
```

Comment:
use this format instead

```suggestion
import {
  StyleSheet,
  Text,
  TextInput,
  View,
  Image,
  TouchableOpacity,
  KeyboardAvoidingView
} from 'react-native';
```